### ISSUE TITLE
Cleaning up the outline of the website

### ISSUE THREAD

**Issue by errai34** [UniverseTBD/universeTBD.github.io#67]
<img width="1031" alt="Screenshot 2024-08-05 at 2 10 57 PM" src="https://github.com/user-attachments/assets/f18e3d28-f7c0-41bd-b5c5-6b934fbcab37">

Good to clean up the pages as Jesse mentioned and keeping it minimal, Research and Products only once. Maybe About can become something like Who we are and Sponsors. Trusting the web dev team

**Comment by jayden626** [comment]
I've changed Products and Research to be individual links, no longer drop down menus.
Updated the text "About us" to "About UniverseTBD". Not sure if you had something different in mind. ("Who we are" seems like the role of People > Participants page)
![image](https://github.com/user-attachments/assets/10403b6f-3d21-427c-bf03-ed66ab1ca8cb)


I also refactored the Navbar to get the links from a file. No more having to mess around with the navbar code mess, should be super easy to add/edit/remove links now
